### ISSUE TITLE
Retrieve code review data

### ISSUE THREAD

**Issue by kbuffardi** [ChicoState/chronicle#13]
When a repo's data is retrieved, it should include records of code reviews:

- [x] Code Reviews including date/time it was submitted, id # for the associated PR, reviewer (username), status [comment, approve, or request changes], description. 
- [x] When exporting, the value in the CSV's `Action` column should be `code_review` and the `Author` column should be the person who submitted the code review.

**Comment by MSS-Shalavritha** [comment]
Worked on this issue today and able to see code reviews are being captured on the UI right now:

[
<img width="1280" alt="Screenshot 2024-05-29 at 10 07 53 PM" src="https://github.com/ChicoState/chronicle/assets/88263283/0e258e91-0aa7-499d-b4d6-a9c71376433e">
](url)

**Comment by kbuffardi** [comment]
@MSS-Shalavritha do you have those changes on a branch? If so, please create a PR.

**Comment by MSS-Shalavritha** [comment]
Hi @kbuffardi, Do you want me to create a new branch?

**Comment by kbuffardi** [comment]
Yes, put all new work on a dedicated feature branch. When it's ready for production, create a pull request and the rest of us will verify and review it.

**Comment by MSS-Shalavritha** [comment]
@kbuffardi , added the code on "code-review-feature" branch and created a pull request. I find the code reviews are being captured and the code review details are being exported to the .csv file as well.
